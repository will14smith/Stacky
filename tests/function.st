main () -> () { 1 mul3 string "a" concat print }

mul3 i64 -> i64 { 3 * }