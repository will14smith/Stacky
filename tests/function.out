=== Output ===
3a

=== Exit ===
0
